autogen/oai/completion.py

```diff
@@ -1067,6 +1077,44 @@ def logged_history(cls) -> Dict:
         """Return the book keeping dictionary."""
         return cls._history_dict
 
+    @classmethod
+    def print_usage_summary(cls) -> Dict:
```

Comment:
Will this function print cost in terms of $? rename this function to `print_token_summary` or `print_cost_summary`?

Reply:
It will print both the total cost in dollar and token summary. Check the example doc.